package.json

```diff
@@ -46,6 +46,11 @@
           "type": "boolean",
           "default": true,
           "description": "Preserve paragraph indent when reflowing lines."
+        },
+        "reflowMarkdown.doubleSpaceBetweenSentences": {
+          "type": "boolean",
+          "default": false,
+          "description": "Insert two spaces instead of one between each sentence."
```

Comment:
It is good that this is an option due to the ongoing debate and personal preference as to whether there should be 1 space or 2.  Thanks!